adapters/unicorn/unicorn.go

```diff
@@ -191,8 +174,9 @@ func setSourceExt() json.RawMessage {
 func setExt(request *openrtb2.BidRequest) (json.RawMessage, error) {
 	accountID, err := jsonparser.GetInt(request.Imp[0].Ext, "bidder", "accountId")
 	if err != nil {
-		accountID = 0
+		return nil, fmt.Errorf("accountId field is required")
```

Comment:
Can you please add coverage for this line?

Reply:
I attemped two pattern.

* accountId is not exists.
* accountId is not integer.

But neither was able to pass this line. I think this is probably because [bidder-params](https://github.com/prebid/prebid-server/blob/master/static/bidder-params/unicorn.json#L19) has an accountId that is integer and required.

Do you have any good ideas?